handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -94,6 +96,20 @@ public SniHandler(AsyncMapping<? super String, ? extends SslContext> mapping) {
     }
 
     /**
+     * Sets if the {@link Channel} should be closed on a handshake failure, default is {@code true}.
+     */
+    public final void setCloseOnHandshakeFailure(boolean closeOnHandshakeFailure) {
+        this.closeOnHandshakeFailure = closeOnHandshakeFailure;
+    }
+
+    /**
+     * Gets if the {@link Channel} should be closed on a handshake failure, default is {@code true}.
+     */
+    public final boolean getCloseOnHandshakeFailure() {
```

Comment:
How about: `setShouldCloseOnHandshakeFailure()` and `shouldCloseOnHandshakeFailure()`?